pricing-page-exercise/saffet/styles.css

```diff
@@ -0,0 +1,257 @@
+body {
+    font-family: sans-serif;
+    margin: 0;
+    padding: 0;
+}
+
+
+/* Navigation bar*/
+.nav-bar{
+    box-sizing: border-box;
+    height: 70px;
+    border: 1px transparent grey;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-left: 25px;
+    padding-right: 35px;
+    box-shadow: rgba(99, 99, 99, 0.2) 0px 2px 8px 0px;
+}
+
+.nav-bar > p{
```

Comment:
I'd not recommend using a selector like this for this case. My suggestion is to make the "Company name" a link element to make it closer to what it would be in real life. Then it would be better to use id/class selector to add styles (my personal preference is to always use classes in CSS, not id).

Reply:
I used class to describe "Company name" as you adviced and I added <a> tag